src/components/FadeTransitioner/SaveForLater__Fader.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
```

Comment:
ignore this file, committing in case we want more control over transitions in the future